front/src/common/Map/Search/useSearchOperationalPoint.tsx

```diff
@@ -55,10 +55,29 @@ export default function useSearchOperationalPoint({
     }
   };
 
-  const sortByMainOperationalPoints = (
+  const sortOperationalPoints = (
     a: SearchResultItemOperationalPoint,
     b: SearchResultItemOperationalPoint
   ) => {
+    const upperCaseSearchTerm = debouncedSearchTerm.toUpperCase();
+    const lowerCaseSearchTerm = debouncedSearchTerm.toLowerCase();
+
+    if (a.trigram === upperCaseSearchTerm && b.trigram !== upperCaseSearchTerm) {
```

Comment:
Maybe you can add some documentation here with comments
```suggestion
    // ops with trigram match first
    if (a.trigram === upperCaseSearchTerm && b.trigram !== upperCaseSearchTerm) {
```